client/src/main/java/com/alibaba/nacos/client/naming/remote/gprc/NamingGrpcClientProxy.java

```diff
@@ -86,7 +86,7 @@ public NamingGrpcClientProxy(String namespaceId, ServerListFactory serverListFac
     private void start(ServerListFactory serverListFactory, ServiceInfoHolder serviceInfoHolder) throws NacosException {
         rpcClient.init(serverListFactory);
         rpcClient.start();
-        rpcClient.registerServerPushResponseHandler(new NamingPushResponseHandler(serviceInfoHolder));
+        rpcClient.registerServerRequestHandler(new NamingPushResponseHandler(serviceInfoHolder));
```

Comment:
registerServerResponseHandler?

Reply:
The handler is to process request from server.

Reply:
Honestly speaking, it is bit counter-intuitive to inject a response handler to a request handler registry.

Reply:
In my opinion. For this case, the client is `requested` side, the server is `request` side. Client received request from server, it process the request and then return reponse to server.